packages/garbo-lib/src/wanderer/lovebugs.ts

```diff
@@ -27,7 +27,7 @@ export function lovebugsFactory(
   _locationSkiplist: Location[],
   options: WandererFactoryOptions,
 ): WandererTarget[] {
-  if (get("lovebugsUnlocked")) {
+  if (get("lovebugsUnlocked") && _type !== "freerun") {
```

Comment:
If we're actually using the variable it shouldn't be preceded by an underscore anymore